packages/appstore/src/components/account/account.tsx

```diff
@@ -1,54 +1,307 @@
+                        account?.server_info?.geolocation?.sequence === 1
+                            ? ''
+                            : account?.server_info?.geolocation?.sequence
+                    }`;
+                }
+            }
+            return '';
+        },
+        []
+    );
+    if (has_error)
+        return (
+            <div>
+                <Text color='disabled' size='xs'>
+                    <Localize i18n_default_text='Unavailable' />
+                </Text>
+                {server?.server_info?.geolocation &&
+                    should_show_server_name &&
+                    market_type === 'synthetic' &&
+                    shortcode === 'svg' && (
+                        <Text color='less-prominent' size='xxs' className='badge-server badge-server--disabled'>
+                            {getServerName(server)}
+                        </Text>
+                    )}
+            </div>
+        );
+    return (
+        <div>
+            {getCFDAccountDisplay({ market_type, sub_account_type, platform, is_eu, shortcode })}
+            {server?.server_info?.geolocation &&
+                should_show_server_name &&
+                market_type === 'synthetic' &&
+                shortcode === 'svg' && (
+                    <Text
+                        color={is_dark_mode_on ? 'general' : 'colored-background'}
+                        size='xxs'
+                        className={classNames('badge-server', {
+                            'badge-server-bot': isBot(),
+                        })}
+                    >
+                        {getServerName(server)}
+                    </Text>
+                )}
+        </div>
+    );
 };
 
 const OptionsAccount = ({
-    account_icon,
-    account_title,
-    account_number,
-    account_balance,
-    account_button,
+    currency_icon,
+    loginid_text,
+    balance,
     currency,
+    display_type,
+    has_balance,
+    has_reset_balance,
+    is_disabled,
+    is_virtual,
+    country_standpoint,
+    is_eu,
+    market_type,
+    server,
+    sub_account_type,
+    has_error,
+    platform,
+    is_dark_mode_on,
+    shortcode,
+    should_show_server_name,
+    onClickResetVirtualBalance,
+    redirectAccount,
+    activeAccount,
+    onClickDeposit,
+    switchAccountModal,
+    isModal,
 }: TOptionsAccountprops) => {
+    const currency_badge = currency ? currency_icon : 'IcCurrencyUnknown';
     return (
-        <div className='account__container'>
-            <div className='account__container--icon'>
-                <WalletIcon icon={account_icon} />
+        <div
+            className={classNames('account-container', {
+                'account-container-active': activeAccount === loginid_text,
+                'account-container-disabled': activeAccount !== loginid_text,
+                'account-container-modal': isModal,
+            })}
+            onClick={activeAccount !== loginid_text ? redirectAccount : undefined}
```

Comment:
Clicking the account component does not select that account immediately and it has lag. before running any logic onClick(checking the loginid_text with activeAccount) , you need to change the className of this component so the user be able to see the background color is changed whenever she/he clicks that account.

Reply:
Discussed with design team, Planning to add a loader for this.